src/QandA/components/AnswerItem.jsx

```diff
@@ -77,7 +78,7 @@ class AnswerItem extends React.Component {
           </div>
         </div>
       <div className="photo-details">
-        <Photos photos={this.props.answer.photos}/>
+        <Photos photos={_.pluck(this.props.answer.photos, 'url')}/>
```

Comment:
Nice one with the underscore pluck. There are other places in the app we could use a similar pattern in passing props. Might be worth mentioning in a group setting.